net-p2p/ktorrent/ktorrent-9999.ebuild

```diff
@@ -56,25 +54,35 @@ COMMON_DEPEND="
 	$(add_qt_dep qtdbus)
 	$(add_qt_dep qtgui)
 	$(add_qt_dep qtnetwork)
+	$(add_qt_dep qttest)
```

Comment:
having set KDE_TEST=optional way above means that you should drop this line here, eclass will do